cilium-dbg/cmd/statedb.go

```diff
@@ -95,6 +95,7 @@ func init() {
 		statedbTableCommand[*tables.Route]("routes"),
 		statedbTableCommand[*tables.L2AnnounceEntry]("l2-announce"),
 		statedbTableCommand[tables.NodeAddress]("node-addresses"),
+		statedbTableCommand[*tables.DesiredQdisc]("bandwidth-manager-qdiscs"),
```

Comment:
Don't quite like `DesiredQdisc` or `bandwidth-manager-qdiscs`. Suggestions welcome for better names.